packages/graphiql-react/src/explorer/components/search.tsx

```diff
@@ -61,124 +53,99 @@ export function Search() {
-                if (!(element instanceof HTMLElement)) {
-                  return;
-                }
-                const top = element.offsetTop - container.scrollTop;
-                const bottom =
-                  container.scrollTop +
-                  container.clientHeight -
-                  (element.offsetTop + element.clientHeight);
-                if (bottom < 0) {
-                  container.scrollTop -= bottom;
-                }
-                if (top < 0) {
-                  container.scrollTop += top;
-                }
-              });
-            }
-
-            // We don't want for example "Escape" key presses to bubble up
-            // further. This could have other effects like closing a dialog
-            // that contains this component.
-            event.stopPropagation();
-          }}
+        <Command.Input
+          autoComplete="off"
+          onFocus={handleFocus}
+          onBlur={handleFocus}
+          onValueChange={setSearchValue}
           placeholder="&#x2318; K"
           ref={inputRef}
           value={searchValue}
         />
       </div>
-      <ComboboxPopover portal={false} ref={popoverRef}>
-        <ComboboxList>
-          {/**
-           * Setting the `index` prop explicitly on the `ComboboxOption` solves
-           * buggy behavior of the internal ordering of the combobox items.
-           * (Sometimes this results in weird jumps when using the keyboard to
-           * navigate search results.)
-           */}
-          {results.within.map((result, i) => (
-            <ComboboxOption key={`within-${i}`} index={i} value={result as any}>
-              <Field field={result.field} argument={result.argument} />
-            </ComboboxOption>
-          ))}
-          {results.within.length > 0 &&
-          results.types.length + results.fields.length > 0 ? (
-            <div className="graphiql-doc-explorer-search-divider">
-              Other results
-            </div>
-          ) : null}
-          {results.types.map((result, i) => (
-            <ComboboxOption
-              key={`type-${i}`}
-              index={results.within.length + i}
-              value={result as any}
-            >
-              <Type type={result.type} />
-            </ComboboxOption>
-          ))}
-          {results.fields.map((result, i) => (
-            <ComboboxOption
-              key={`field-${i}`}
-              index={results.within.length + results.types.length + i}
-              value={result as any}
-            >
-              <Type type={result.type} />.
-              <Field field={result.field} argument={result.argument} />
-            </ComboboxOption>
-          ))}
-          {results.within.length +
-            results.types.length +
-            results.fields.length ===
-          0 ? (
-            <div className="graphiql-doc-explorer-search-empty">
-              No results found
-            </div>
-          ) : null}
```

Comment:
cmdk exports abstract `<Command.Empty />` so this code was removed and replace by 

```ts
         <Command.Empty className="graphiql-doc-explorer-search-empty">
            No results found
          </Command.Empty>
```